disassemblers/ofrak_pyghidra/ofrak_pyghidra/components/pyghidra_components.py

```diff
@@ -0,0 +1,186 @@
+from dataclasses import dataclass
+import json
+from tempfile import TemporaryDirectory
+import os
+from typing import Dict
+
+from ofrak.component.analyzer import Analyzer
+from ofrak.service.data_service_i import DataServiceInterface
+from ofrak.service.resource_service_i import ResourceFilter, ResourceServiceInterface
+from ofrak_type import ArchInfo, Endianness, InstructionSet
+
+
+from ofrak.component.identifier import Identifier
+from ofrak.core.elf.model import Elf
+from ofrak.core.ihex import Ihex
+from ofrak.core.pe.model import Pe
+from ofrak.core.program import Program
+from ofrak.model.component_model import ComponentConfig
+from ofrak.resource import Resource, ResourceFactory
+from ofrak.resource_view import ResourceView
+from ofrak_cached_disassembly.components.cached_disassembly import CachedAnalysisStore
+from ofrak_cached_disassembly.components.cached_disassembly_unpacker import (
+    CachedCodeRegionUnpacker,
+    CachedComplexBlockUnpacker,
+    CachedBasicBlockUnpacker,
+    CachedCodeRegionModifier,
+    CachedDecompilationAnalyzer,
+)
+from ofrak_pyghidra.standalone.pyghidra_analysis import unpack
+
+
+_GHIDRA_AUTO_LOADABLE_FORMATS = [Elf, Ihex, Pe]
+
+
+@dataclass
+class PyGhidraAutoLoadProject(ResourceView):
+    pass
+
+
+@dataclass
+class PyGhidraProject(ResourceView):
+    pass
```

Comment:
Similar comment to the cached unpackers: should this be resource attributes instead of a resource view?